src/static/css/footer.less

```diff
@@ -164,7 +168,7 @@
             margin: 0;
         }
 
-        .respond-to-range(768px, 1199px,{
+        .respond-to-min(@tablet-min {
```

Comment:
The decision was to move to something that worked on smaller desktop screens to reduce the need for multiple breakpoints. The older design runs into problems at a smaller breakpoint and if we want to add longer navigations items there.

 /flapjack/global/issues/10#issuecomment-48108